torch/export/exported_program.py

```diff
@@ -122,10 +122,10 @@ def __init__(
         self._graph_signature: ExportGraphSignature = graph_signature
         self._state_dict: Dict[str, Any] = state_dict
         self._range_constraints: "Dict[sympy.Symbol, ValueRanges]" = range_constraints
-        self._equality_constraints: List[
-            Tuple[InputDim, InputDim]
-        ] = equality_constraints
-        self._module_call_graph: List[ModuleCallEntry] = module_call_graph
+        self._equality_constraints: List[Tuple[InputDim, InputDim]] = (
+            equality_constraints or []
+        )
+        self._module_call_graph: List[ModuleCallEntry] = module_call_graph or []
```

Comment:
Suggestion: If it's too hard to make it not optional for now, better enforce the invariants in code directly.
```suggestion
        assert module_call_graph is not None
        self._module_call_graph: List[ModuleCallEntry] = module_call_graph
        assert len(self._module_call_graph) >= 1
```